packages/api-client/src/odsql/index.ts

```diff
@@ -114,16 +129,17 @@ export class Query {
 function root(source: string) {
     return () => ({
         itself: () => new Query(`${source}/`),
-        aggregates: () => new Query(`${source}/aggregates/`),
         facets: () => new Query(`${source}/facets/`),
         datasets: () => new Query(`${source}/datasets/`),
+        query: () => new Query(`${source}/query/`),
+        exports: (format: string) => new Query(`${source}/exports/${format}/`),
         dataset: (datasetId: string) => ({
             itself: () => new Query(`${source}/datasets/${datasetId}/`),
-            aggregates: () => new Query(`${source}/datasets/${datasetId}/aggregates/`),
             facets: () => new Query(`${source}/datasets/${datasetId}/facets/`),
             records: () => new Query(`${source}/datasets/${datasetId}/records/`),
+            query: () => new Query(`${source}/datasets/${datasetId}/query/`),
```

Comment:
Note: in query we need to pass `&include_metas=True` to get hateoas links

Reply:
I discussed with the api teams and we it is better to wait for those feature to be publicly documented first. Hateoas link don't really matter here yet, and are available in records/datasets endpoints.